.devcontainer/devcontainer.json

```diff
@@ -1,4 +1,9 @@
 {
+  "hostRequirements": {
+    "cpus": 4,
+    "memory": "8gb",
+    "storage": "32gb"
```

Comment:
Do we actually use that much? Or it's just so that we have plenty in future if running `yarn start ...` ?

Reply:
I think 32 GB or storage is the default for the 4 core tier?